src/webgpu/api/operation/memory_sync/texture/texture_sync_test.ts

```diff
@@ -175,8 +175,51 @@ export function checkOpsValidForContext(
   ops: [Op, Op],
   context: [OperationContext, OperationContext]
 ) {
-  return (
+  const valid =
     kOpInfo[ops[0]].contexts.indexOf(context[0]) !== -1 &&
-    kOpInfo[ops[1]].contexts.indexOf(context[1]) !== -1
-  );
+    kOpInfo[ops[1]].contexts.indexOf(context[1]) !== -1;
```

Comment:
nit: `.includes` is more readable
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes